ILRepack/Steps/ModuleInitializersRepackStep.cs

```diff
@@ -135,9 +135,26 @@ private void DemoteModuleInitializerMethodToNormalMethod(MethodDefinition initia
             initializer.IsRuntimeSpecialName = false;
         }
 
+        private Dictionary<string, AssemblyDefinition> ToDictionarySkipDuplicates(IEnumerable<AssemblyDefinition> assemblies)
+        {
+            var dict = new Dictionary<string, AssemblyDefinition>();
+            foreach (var assembly in assemblies)
+            {
+                if (!dict.ContainsKey(assembly.Name.Name))
```

Comment:
assembly.Name.Name is repeated three times, it's good to extract a local variable and reuse it